test/bug_patches/spm_file_template.m

```diff
@@ -0,0 +1,204 @@
+classdef spm_file_template
```

Comment:
Could we patch on-the-fly as suggested by Guillaume in https://github.com/incf-nidash/nidmresults-spmhtml/pull/38#issuecomment-389840670?

Reply:
Oh! Right! I had misunderstood here, yes I will change this! I thought this was intended to go in the docker image eventually.

Reply:
Hopefully it's only temporary as SPM dev includes the workaround and there is a [patch submitted](https://savannah.gnu.org/bugs/?53924) for Octave.